stakbucks/230704/b_9020_골드바흐의추측/app.js

```diff
@@ -0,0 +1,33 @@
+const fs = require('fs');
+const filePath = process.platform === 'linux' ? '/dev/stdin' : './input.txt';
+let input = fs.readFileSync(filePath).toString().trim().split('\n');
+
+function solution(input) {
+  const test = input.splice(1).map(Number);
+
+  const answer = [];
+  // 소수 여부 판별 함수
+  function isPrime(num) {
+    for (let i = 2; i * i <= num; i++) {
+      if (num % i === 0) {
+        return false;
+      }
+    }
+    return true;
+  }
+  for (const num of test) {
+    let a = num / 2;
+    let b = num / 2;
+    while (1) {
+      if (isPrime(a) && isPrime(b)) {
+        answer.push([a, b].join(' '));
```

Comment:
```suggestion
        console.log(a, b);
        break;
```
와 같이 처리하면 answer 변수 선언을 패스할 수 있을 것 같아요!

Reply:
console.log는 디버깅용 함수라 시간을 많이 잡아먹어서, 반복문안에 넣어서 정답을 여러번 나눠서 출력하면 성능저하가 큰 걸로 알고 있습니다. 실제로 백준에서도 console.log를 여러번 사용하는 게 훨씬 채점 속도가 느리고, 몇몇 문제는 아예 시간초과가 납니다. 그래서 정답을 answer변수에 담아서 한번에 출력했습니다!